config/vlc.py

```diff
@@ -0,0 +1,5660 @@
+#! /usr/bin/python
```

Comment:
It looks like this library can be installed with pip via:  `pip install python-vlc`. Could we install that and just add the modified lines under the `if __name__ == '__main__':` to a simpler script? This would make it a lot easier to keep up to date with the latest vlc version.

Reply:
That sounds like a very reasonable option.